spec/features/visitors/sign_up_spec.rb

```diff
@@ -32,11 +32,6 @@
   scenario 'visitor can sign up and confirm a valid email' do
     sign_up_with('test@example.com')
 
-    stub_analytics
```

Comment:
They don't :) The analytics tests were moved from feature tests to controller tests, where they belong.